sw/host/opentitanlib/src/transport/ti50emulator/spi.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::io::spi::{SpiError, Target, Transfer, TransferMode};
+use crate::transport::TransportError;
+use crate::util::voltage::Voltage;
+use anyhow::Result;
```

Comment:
style nit: `use anyhow::Result` should be first (generally, extern and std crates first, then local `crate::`s).